app/views/directives/deploy-image.html

```diff
@@ -196,6 +200,7 @@ <h2>
         Could not load image metadata.
       </h2>
       <p>{{import.error | upperFirst}}</p>
+      <p>The image may not exist or it may be in a secure registry. Check that you have entered the image name correctly or <a href="" ng-click="openCreateWebhookSecretModal()">create an image pull secret</a> from your Docker credentials and try again.</p>
```

Comment:
"...with your image registry credentials and try again."

Reply:
updated